src/index.js

```diff
@@ -0,0 +1,20 @@
+import './style.css';
+
+const todoTasks = [
+  { index: 1, type: 'Code Development', completed: true },
+  { index: 2, type: 'Code Testing', completed: false },
+  { index: 3, type: 'Code Maintenance', completed: true },
+];
+
+const todoHolder = document.querySelector('');
+
+const todos = todoTasks.map((todo) => `
+  <div class="todoTasks">
+    <div class="task"><input type="checkbox" id="checkBox" name="checkbox" value="1">
+      <label for="checkBox">${todo.type}</label>
+    </div>
+    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
+  </div>
+`).join('');
+
```

Comment:
- [ ] You did great so far, but I think there are some issues you need to fix before moving to the next milestone, 
- [ ] Looking at the video, you can see that your application starts up with a webpack compilation issue with reference to the `mode`, even after closing the error, you can it gives an error in the console and the todo items are not shown in the UI, I'll suggest you rectify this issue. :+1: :+1: :100: 

- - VIDEO REFERENCE:

https://github.com/Kgomotso196/To-Do-List/assets/107632780/3df5ddf0-84d0-4c6e-a6df-676953df4ab2